_posts/en/posts/2024-06-10-disclose_upnp_rce.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Disclosure of potential remote code execution due to bug in miniupnpc (&le; version 0.11.1)
+name: blog-disclose-upnp-rce
+id: en-blog-disclose-upnp-rce
+lang: en
+type: advisory
+layout: post
+
+## If this is a new post, reset this counter to 1.
+version: 1
+
+## Only true if release announcement or security annoucement. English posts only
+announcement: 1
+
+excerpt: >
+  Public disclosure of a buffer overflow in miniupnpc which could have led to a remote code execution in Bitcoin Core.
+---
+
+A buffer overflow enabling a significant data leak was discovered in `miniupnpc`. Combined with the then
+recently-disclosed `CVE-2015-6031` it enabled an RCE in `miniupnpc` which could have led to an RCE
```

Comment:
style nit: I find that putting CVE numbers in code blocks is a bit hard to read. Our existing disclosures don't do that either.

Reply:
Done.